toposort/meta.yaml

```diff
@@ -11,7 +11,7 @@ source:
    # - fix.patch
 
 build:
-  noarch_python: False
+  noarch: True
```

Comment:
just a note that this should be `noarch: False`. True will build a package which installs but ModuleNotFound's on import